acrobat/scripts/contentSecurityPolicy/prod.js

```diff
@@ -44,6 +44,30 @@ const connectSrc = [
   '*.typekit.net/',
   'trial-eum-clienttons-s.akamaihd.net/',
   '*.akstat.io/',
+  '345-tti-184.mktoutil.com',
+  'adservice.google.com',
+  'rum.hlx.page',
+  'us-central1-adaptive-growth.cloudfunctions.net',
+  '360-kci-804.mktoresp.com',
+  'analytics.google.com',
+  'api.ultimateadb.com',
+  'c.clicktale.net',
+  'cdnmd.global-cache.online',
+  'ff.kis.v2.scr.kaspersky-labs.com',
```

Comment:
do we deliberately put kaspersky-labs on our page, or is this just something we see reject in the field (because people have anti-virus software installed)?  If the latter, we probably do NOT want it on the allow list

Reply:
did security review this list?

Reply:
we can speak w/ security